react/react.d.ts

```diff
@@ -132,6 +132,11 @@ declare namespace __React {
 
     // Base component for plain JS classes
     class Component<P, S> implements ComponentLifecycle<P, S> {
+        static propTypes: ValidationMap<any>;
+        static contextTypes: ValidationMap<any>;
+        static childContextTypes: ValidationMap<any>;
+        static defaultProps: Props<any>;
```

Comment:
It's a shameless copy-paste from ComponentClass, since there is no way (that I know of) to link it to Component.